### PR TITLE
Bug 1214533 - Moves clear history and cookies buttons. Refactors conf…

### PR DESCRIPTION
**Pull request by scottwu**
…irm form with dialog service. Adds integration tests. r=gasolin

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=33320536ded3a7b0a07d078c739e64b1cb702187

File: apps/settings/js/panels/browsing_privacy/panel.js [hunk 1]
```diff
@@ -3,18 +3,12 @@ define(function(require) {
 
   var SettingsPanel = require('modules/settings_panel');
   var BrowsingPrivacy = require('panels/browsing_privacy/browsing_privacy');
+  var DialogService = require('modules/dialog_service');
```
Comment (gasolin) [inline apps/settings/js/panels/browsing_privacy/panel.js]: remember to exclude `modules/dialog_service` in `js/config/require.js` when you use it

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f9b73545da163c00722dcc67ac57a1faf62df8ce

File: apps/settings/js/config/require.js [hunk 2]
```diff
@@ -303,7 +303,10 @@ require.config({
     },
     {
       name: 'panels/browsing_privacy/panel',
-      exclude: ['main']
+      exclude: [
+        'main',
+        'modules/dialog_service'
```
Comment (scottwu) [inline apps/settings/js/config/require.js]: excluded dialog_service

File: apps/settings/js/panels/browsing_privacy/panel.js (hunk 1 above)
Comment (scottwu) [inline apps/settings/js/panels/browsing_privacy/panel.js]: got it. thanks for the catch

**Pull request merged**